File: src/main/java/olivermakesco/de/refmagic/registry/RefinedMagicRegistry.java
package olivermakesco.de.refmagic.registry;

import org.quiltmc.qsl.item.group.api.QuiltItemGroup;

import static olivermakesco.de.refmagic.Mod.id;

public class RefinedMagicRegistry {
    public static QuiltItemGroup group = QuiltItemGroup.builder(id("items")).appendItems(RefinedMagicPotions::appendStacks).icon(() -> RefinedMagicItems.kyriteShard.getDefaultStack()).build();
    public static void register() {
        RefinedMagicPotions.register();
        RefinedMagicBlocks.register();
        RefinedMagicBlockEntities.register();
        RefinedMagicItems.register();
        RefinedMagicWorldgen.register();
        RefinedMagicRecipes.register();
    }
}

File: src/main/kotlin/olivermakesco/de/refmagic/registry/RefinedMagicRegistry.kt
package olivermakesco.de.refmagic.registry

import olivermakesco.de.refmagic.Mod
import org.quiltmc.qsl.item.group.api.QuiltItemGroup

object RefinedMagicRegistry {
    var items: QuiltItemGroup = QuiltItemGroup.builder(Mod.id("items"))
        .appendItems(RefinedMagicPotions::appendStacks)
        .icon { RefinedMagicItems.kyriteShard.defaultStack }
        .build()

    fun register() {
        RefinedMagicPotions.register()
        RefinedMagicBlocks.register()
        RefinedMagicBlockEntities.register()
        RefinedMagicItems.register()
        RefinedMagicWorldgen.register()
        RefinedMagicRecipes.register()
    }
}